src/main/java/jdbchomework/utils/InjectorHibTest.java

```diff
@@ -0,0 +1,54 @@
+package jdbchomework.utils;
+
+import jdbchomework.console.ConsoleMain;
+import jdbchomework.console.ContentsUserMenu;
+import jdbchomework.console.VisualUserMenu;
+import jdbchomework.controller.CompanyController;
+import jdbchomework.controller.CustomerController;
+import jdbchomework.controller.DeveloperController;
+import jdbchomework.controller.ProjectController;
+import jdbchomework.controller.SkillController;
+import jdbchomework.dao.hibernate.CompaniesHibDao;
+import jdbchomework.dao.hibernate.CustomersHibDao;
+import jdbchomework.dao.hibernate.DevelopersHibDao;
+import jdbchomework.dao.hibernate.ProjectsHibDao;
+import jdbchomework.dao.hibernate.SkillsHibDao;
+import jdbchomework.entity.Company;
+import jdbchomework.entity.Customer;
+import jdbchomework.entity.Developer;
+import jdbchomework.entity.Project;
+import jdbchomework.entity.Skill;
+import jdbchomework.service.CompanyService;
+import jdbchomework.service.CustomerService;
+import jdbchomework.service.DeveloperService;
+import jdbchomework.service.ProjectService;
+import jdbchomework.service.SkillService;
+import org.hibernate.SessionFactory;
+
+public class InjectorHibTest {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)